src/components/objects/Player.js

```diff
@@ -11,14 +11,11 @@ class Player extends Phaser.Group {
     constructor(data, game) {
         super(game, null, 'Player', true, true);
 
-        // this._direction = game..directions.RIGHT;
-        this._moveTimer = 0;
-
         for (const bodyPart of data._bodyParts) {
             this.expandBody({
                 x: bodyPart._x,
                 y: bodyPart._y,
-            });
+            }, data._color);
```

Comment:
I dont know why, but it feels like I wanna say `this._color` or `player._color` but we dont have a reference to the player here right?